packages/client/public/index.html

```diff
@@ -1,26 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-     <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-WCDTMFM6RG"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
 
-  gtag('config', 'G-WCDTMFM6RG');
-</script>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title><%= htmlWebpackPlugin.options.title %></title>
-    <link rel="stylesheet" href="https://unpkg.com/vue-multiselect@2.1.0/dist/vue-multiselect.min.css">
-  </head>
-  <body>
-    <noscript>
-      <strong>We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
-    </noscript>
-    <div id="app"></div>
-  </body>
+<head>
+  <script src="/deploy-config.js"></script>
```

Comment:
Where is this file defined? I don't see it in the repository anywhere or this PR. Am I missing something?

Reply:
Yep, the file is not defined in this repository because its contents are specific to the deployment environment; its absence is what enables builds that are agnostic to their runtime configuration. 

I considered adding a "dummy" version of the file defined in the repo, which would simply get overwritten at deploy-time, but I couldn't think of a strong argument for doing so and worried it would be even more confusing to serve a different file than what's tracked in the repo.

Reply:
Noted! This PR auto-merged, but I will include an example file as a side-addition in my upcoming PR to adds a website build pipeline. Thanks for the suggestion 👍